galsim/lsst/lsst_wcs.py

```diff
@@ -0,0 +1,1153 @@
+import numpy as np
+import warnings
+import galsim
+
+try:
+    import lsst.pex.logging as pexLog
+    import lsst.daf.base as dafBase
+    import lsst.afw.geom as afwGeom
+    import lsst.afw.cameraGeom as cameraGeom
+    import lsst.afw.image as afwImage
+    import lsst.afw.image.utils as afwImageUtils
+    import lsst.meas.astrom as measAstrom
+    from lsst.afw.cameraGeom import PUPIL, PIXELS, TAN_PIXELS, FOCAL_PLANE
+    from lsst.obs.lsstSim import LsstSimMapper
+except ImportError:
+    raise ImportError("You cannot use the LSST module.\n"
+                      "You either do not have the LSST stack installed,\n"
+                      "or you have it installed, but have not set it up.\n"
+                      "------------\n"
+                      "To install the LSST stack, follow the instructions at:\n\n"
+                      "https://confluence.lsstcorp.org/display/SIM/Catalogs+and+MAF\n\n"
+                      "NOTE: you must build the stack with the python you are using to\n"
+                      "run GalSim.  This means that when the stack asks you if you want\n"
+                      "it to install Anaconda for you , you MUST SAY NO.\n\n"
```

Comment:
I didn't say no, and I didn't have any trouble in that respect.  I'd suggest removing this sentence.  It just means that the python you need to use for using galsim.lsst is the miniconda python that LSST installs.  That seems fine and probably the easiest option for most users.

Also, I updated the [GalSim wiki page](https://github.com/GalSim-developers/GalSim/wiki/Building-GalSim-with-the-LSST-stack) about building with the stack, so perhaps that would make a better link for this error message.